train_segmentation.py

```diff
@@ -112,7 +112,7 @@ def create_dataloader(samples_folder, batch_size, num_devices, params):
                                 drop_last=True)
     # Using batch_metrics with shuffle=False on val dataset will always mesure metrics on the same portion of the val samples.
```

Comment:
a) adjust the two following commented lines to reflect why it should be set to shuffle=False  
or  
b) delete those two commented lines.